README.md

```diff
@@ -29,15 +29,15 @@ Getting started with the CoVID-19 CLI is very simple. Kindly follow the steps gi
 
 3.  Create a symlink of the repository in the global folder
 
-- Windows
-  If you are on Windows, start `cmd.exe` as Administrator and run the command
+- **Windows**  
```

Comment:
You need to indent these bullet points so that it comes directly under the third point (given above).

Reply:
Alright. I finally get what is to be done.
I have now made them sub-bullets under Point 3.